### ISSUE TITLE
Make docs fails on 1.0.4

### ISSUE THREAD

**Issue by jaseemabid** [ninenines/cowboy#1002]
```
$  git log -1 --oneline
d08c2ab Update to 1.0.4

$ make docs

 GEN    distclean-asciidoc
 GEN    distclean-edoc
 GEN    edoc
./src/cowboy_protocol.erl: at line 273: syntax error before: 'end'
edoc: skipping source file './src/cowboy_protocol.erl': {'EXIT',error}.
edoc: error in doclet 'edoc_doclet': {'EXIT',error}.
{"init terminating in do_boot",error}

Crash dump is being written to: erl_crash.dump...done
init terminating in do_boot (error)
make: *** [edoc] Error 1
```

**Comment by essen** [comment]
Not using edoc so probably a mistake that it tries to run it. Thanks!

**Comment by essen** [comment]
Can't reproduce on the soon to be released 1.1.0 version. Reopen with more info if it still happens then. Thanks!
